frontend/src/components/favoriteEventsPage/CardGridFavoriteEvents.jsx

```diff
@@ -1,22 +1,17 @@
-import EventCard from "../EventCard.jsx";
-import data from "../../data.js";
+import CardEvent from "../CardEvent.jsx";
+import data from "../../data.js"
 
-const CardGridFavoriteEvents = () => {
-  return (
-    <div className="grid lg:grid-cols-4 sm:grid-cols-2 ">
-      {data.map((d) => (
-        <div className="m-[3rem]">
-          <EventCard
-            name={d.name}
-            location={d.location}
-            imageUrl={d.imageUrl}
-            score={d.score}
-            numReviews={d.numReviews}
-          />
-        </div>
-      ))}
-    </div>
-  );
-};
 
-export default CardGridFavoriteEvents;
+const CardGridFavoriteEvents = () => {
+    return (
+      <div className="grid lg:grid-cols-4 sm:grid-cols-2 ">
+        {data.map((d) => (
+            <div className="m-[3rem]">
+              <CardEvent data={d} />
+            </div>
+        ))}
+      </div>
+    );
+  };
+  
+  export default CardGridFavoriteEvents;
```

Comment:
Stretch goal here: This is basically the same as CardGridForYou. Consider making CardGridEvents as a UI component, and allow FavoriteEventsPage and ForYouPage to leverage it by passing in different dataset.

Reply:
**_To do:_**
Refactor code to not use repeated code